storage/service_account/client_endpoint/client_endpoint_test.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package clientendpoint
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+func TestSetClientEndpoint(t *testing.T) {
+	tc := testutil.SystemTest(t)
```

Comment:
I don't think we need this line, because the ProjectID does not get used in the `setClientEndpoint()` method, making this more of a unit test than a "system test".

Reply:
@muncus I used testutil based on the [guide](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/CONTRIBUTING.md#use-testutil-for-tests). In this case, do you recommend to remove the testutil and skip tests instead?

Reply:
I hesitate to suggest less testing 😅 but my understanding here is that testutil is not needed, because the test code does not actually call the Storage service, and thus no Project ID is actually needed.
That's my perspective, but I'll tag in @enocom for a second opinion.